### PR TITLE
Improve documentation of parameters

### PR DESCRIPTION
**Pull request by kwin**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MSCMPUBLISH) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MSCMPUBLISH-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MSCMPUBLISH-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify -Prun-its` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Review by michael-o — CHANGES_REQUESTED** [review]
One nit

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 ${project.name}
 
     The maven-scm-publish-plugin is a utility plugin to allow publishing Maven
-    website to any supported SCM.
+    website to any supported SCM (Source Code Management system like Git).
```
[the remark this answers is not in the document]
Comment (kwin) [reply src/site/apt/index.apt.vm:32]: Good point, fixed now.

**Review by michael-o — APPROVED** [review]

File: src/main/java/org/apache/maven/plugins/scmpublish/AbstractScmPublishMojo.java [hunk 2]
```diff
@@ -107,13 +110,13 @@ public abstract class AbstractScmPublishMojo extends AbstractMojo {
     protected String subDirectory;
 
     /**
-     * Display list of added, deleted, and changed files, but do not do any actual SCM operations.
+     * If set to {@true} displays list of added, deleted, and changed files, but does not do any actual SCM operations.
```
Comment (pzygielo) [inline src/main/java/org/apache/maven/plugins/scmpublish/AbstractScmPublishMojo.java:113]: Perhaps `{@code true}`?

Comment (kwin) [reply src/main/java/org/apache/maven/plugins/scmpublish/AbstractScmPublishMojo.java:113]: Indeed, fixed now in https://github.com/apache/maven-scm-publish-plugin/commit/ecf4fbfa2a88f673c5b9a86b1171bf2d6972278a.

**Pull request merged**